atlasdb-commons/src/main/java/com/palantir/util/AssertUtils.java

```diff
@@ -26,7 +26,18 @@
 
 public class AssertUtils {
 
-    private static final Logger log = LoggerFactory.getLogger(AssertUtils.class);
+    /**
+     * Previously, failed asserts logged to this log.
+     * However, this led to the problem that the logger would always be com.palantir.util.AssertUtils,
+     * which is extremely annoying to try to filter based on your logging properties.
+     * (Should it go into the server error log, or maybe the upgrade log, or the import log file?
+     * Can't tell, cause it's all AssertUtils!)
+     *
+     * Until we get all downstream projects off of using defaultLog however,
+     * this will stay, just deprecated.
+     */
+    @Deprecated
+    private static final Logger defaultLog = LoggerFactory.getLogger(AssertUtils.class);
```

Comment:
Isn't the primary purpose of these helpers to make java "asserts" to catch bugs while running tests, but then still log in case you are in production where asserts are disabled? Sounds like something you would use to catch non-fatal bugs in your logic, rather than do general logging. If you just want to log some irregular condition that is not a bug in your code, why not just write
```
if (!condition) {
   log.error("my message");
}
```
This also lets you set the desired logging level.